Submitted By: Mario Fetka (mario-fetka at gmx dot at)
Date: 2008-07-18
Initial Package Version: 5.11.08
Origin: me
Upstream Status: unknown
Description: correct arm dir
 
diff -Naur makefile.orig /makefile
--- makefile.orig	2007-10-29 16:38:11.000000000 +0000
+++ makefile	2008-07-18 18:42:52.000000000 +0000
@@ -8,7 +8,7 @@
 LIBS =
 ODIR = x86
 ODIR64 = x64
-ODIRARM = arm
+ODIRARM = n800
 
 _OBJS = fileio.o linuxserio.o lowlevel.o server.o errormessage.o flashrom.o webserver.o xap.o lanio.o ascii.o
 OBJS = $(patsubst %,$(ODIR)/%,$(_OBJS))
